src/client/app/actions/monaco.ts

```diff
@@ -27,7 +27,7 @@ export class MonacoActionTypes {
 export class ChangeTabAction implements Action {
     readonly type = MonacoActionTypes.CHANGE_TAB;
 
-    constructor(public payload: string, public language: string) { }
+    constructor(public payload: { name: string, language: string }) {}
```

Comment:
@WrathOfZombies, let me go ahead and merge it in as is.  Ultimately I don't think it matters (it works in my version, and didn't in the original, so it's still a net gain; and you have some other places where `payload` takes in an object anyway).  If you want to go ahead and explore it for academic curiosity, by all means!  But I think I may as well merge it in for the time being.